cms/envs/common.py

```diff
@@ -387,6 +387,19 @@
     # .. toggle_status: supported
     # .. toggle_warnings: None
     'ENABLE_ORA_USER_STATE_UPLOAD_DATA': False,
+
+    # .. toggle_name: SHOW_STUDIO_WARNING_FOR_DEPRECATED_COURSE_KEYS
+    # .. toggle_implementation: DjangoSetting
+    # .. toggle_default: True
+    # .. toggle_description: Toggle for to warn about removing support for deprecated course keys.
+    # .. toggle_category: n/a
+    # .. toggle_use_cases: incremental_release
+    # .. toggle_creation_date: 2020-06-12
+    # .. toggle_expiration_date: 2020-12-31
```

Comment:
TODO: What should the expiration be? I just picked the end of the year (~6 months) on a whim...